.github/actions/baseline-test/action.yml

```
@@ -0,0 +1,222 @@
+name: 'Hello World'
+description: 'Greet someone and record the time'
+inputs:
+  node-version:
+    description: 'node-version'
+    required: true
+    default: '16.x'
+  python-version: 
+    description: python-version
+    default: '3.7'
+    required: true
+  case-path: 
+    description: 'case-path'
+    required: true
+    default:  'success/case1'
+  node_env:
```

keep consistent naming strategy

```suggestion
  node-env:
```

---

All parameters are in underscore format